src/bbd/gis/utils.py

```diff
@@ -112,6 +112,25 @@ def get_geojson_bounds(geojson: dict):
         )
 
 
+def resolve_shapefile_path(in_path):
+    """Resolves to a shapefile path.
```

Comment:
Should include example in docstring of what the paths might look like.. e.g. if folder structure looked like x, and in_path was y, would return z

Reply:
Addressed in 2196520